jerry-core/ecma/builtin-objects/ecma-builtin-json.cpp

```diff
@@ -1198,23 +1198,24 @@ ecma_builtin_json_quote (ecma_string_t *string_p) /**< string that should be quo
   ecma_string_t *product_str_p = ecma_copy_or_ref_ecma_string (quote_str_p);
   ecma_string_t *tmp_str_p;
 
-  ecma_length_t string_len = ecma_string_get_length (string_p);
+  ecma_length_t string_size = ecma_string_get_size (string_p);
```

Comment:
I see you changed length to size here. I think you should do this above as well.